media-libs/quirc/quirc-1.2-r1.ebuild

```diff
@@ -21,6 +21,7 @@ fi
 DEPEND="
 	media-libs/libjpeg-turbo:=
 	media-libs/libpng:=
+	media-libs/libsdl:=
```

Comment:
IMO the optionality of libsdl really should be noted in a comment but I won't block merging over this.